locations/settings.py

```diff
@@ -149,6 +149,19 @@
     "DOWNLOADER_MIDDLEWARES": {"locations.middlewares.playwright_middleware.PlaywrightMiddleware": 543},
 }
 
+DEFAULT_PLAYWRIGHT_SETTINGS_WITH_EXT_JS = {
```

Comment:
Can probably be something like:
```python
DEFAULT_PLAYWRIGHT_SETTINGS_WITH_EXT_JS = DEFAULT_PLAYWRIGHT_SETTINGS | { "PLAYWRIGHT_ABORT_REQUEST": lambda request: not request.resource_type == "document"
    and not request.resource_type == "script"}
```

Reply:
Thanks for the review! I had tried `DEFAULT_PLAYWRIGHT_SETTINGS.update({..})` without success and totally forgot about `|` which is working fine with `from location.settings import DEFAULT_PLAYWRIGHT_SETTINGS_WITH_EXT_JS`.